samples/asgardeo-choreo-react-express/apps/client/src/components/AuthenticationResponse.tsx

```diff
@@ -0,0 +1,155 @@
+/**
```

Comment:
Can we update the year of license header?

Reply:
Fixed with https://github.com/asgardeo/asgardeo-auth-react-sdk/pull/132/commits/1b7642f40cf5c306b95adddf4b6b0f70d3bc462f